velox/common/io/CachedBufferedInput.cpp

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-#include "velox/dwio/common/CachedBufferedInput.h"
+#include "velox/common/io/CachedBufferedInput.h"
 #include "velox/common/memory/Allocation.h"
 #include "velox/common/process/TraceContext.h"
 #include "velox/dwio/common/CacheInputStream.h"
```

Comment:
You will need to move all dependencies (including `ReaderOptions`) to common/io as well, since it should not depend on anything in dwio